app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -22,10 +22,9 @@ class Admin::AdminSetsController < ApplicationController
     self.admin_set_create_service = AdminSetCreateService
 
     def show
-      add_breadcrumb t(:'hyrax.controls.home'), root_path
-      add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), hyrax.dashboard_path
-      add_breadcrumb t(:'hyrax.admin.sidebar.admin_sets'), hyrax.admin_admin_sets_path
-      add_breadcrumb t(:'hyrax.admin.admin_sets.show.breadcrumb'), request.path
```

Comment:
Should this be removed from the translation files (all of them)?

Reply:
Good idea - Cleaned up.